circuits/lib/utils/valueHasher.circom

```diff
@@ -0,0 +1,59 @@
+pragma circom 2.0.0;
+
+include "../../../node_modules/circomlib/circuits/poseidon.circom";
+
+// Because of the way the Poseidon hash function is implemented for Solidity, the max number of inputs must be 6
+template ValueHasher(valueArraySize) {
```

Comment:
May be `spongeHash` and `arraySize`? Too specific naming for the util template

Reply:
renamed